recipes/glib/all/conanfile.py

```diff
@@ -252,11 +243,11 @@ def package_info(self):
             self.cpp_info.components["gio-2.0"].system_libs.extend(["iphlpapi", "dnsapi", "shlwapi"])
             self.cpp_info.components["gio-windows-2.0"].set_property("pkg_config_name", "gio-windows-2.0")
             self.cpp_info.components["gio-windows-2.0"].requires = ["gobject-2.0", "gmodule-no-export-2.0", "gio-2.0"]
-            self.cpp_info.components["gio-windows-2.0"].includedirs = [os.path.join("include", "gio-win32-2.0")]
+            self.cpp_info.components["gio-windows-2.0"].includedirs += [os.path.join("include", "gio-win32-2.0")]
         else:
             self.cpp_info.components["gio-unix-2.0"].set_property("pkg_config_name", "gio-unix-2.0")
             self.cpp_info.components["gio-unix-2.0"].requires += ["gobject-2.0", "gio-2.0"]
-            self.cpp_info.components["gio-unix-2.0"].includedirs = [os.path.join("include", "gio-unix-2.0")]
+            self.cpp_info.components["gio-unix-2.0"].includedirs += [os.path.join("include", "gio-unix-2.0")]
```

Comment:
What is the content of official `gio-windows-2.0.pc` & `gio-unix-2.0 pc` files?

Reply:
`gio-unix-2.0.pc` is

```
prefix=/usr
libdir=${prefix}/lib/x86_64-linux-gnu
includedir=${prefix}/include

Name: GIO unix specific APIs
Description: unix specific headers for glib I/O library
Version: 2.72.1
Requires: gobject-2.0, gio-2.0
Cflags: -I${includedir}/gio-unix-2.0
```

Reply:
Therefore glib recipe was since official include dir of gio-unix-2.0 is `${prefix}/include/gio-unix-2.0`

Reply:
`gobject-introspection` fails because it uses the `includedir` variable. The conan generated `gio-unix-2.0.pc` file is

```
prefix=/home/andrew/.conan/data/glib/2.73.3/_/_/package/eb34e48f59e3766360e3256bde88ef47a36ecfe0
libdir1=${prefix}/lib
includedir1=${prefix}/include/gio-unix-2.0

Name: gio-unix-2.0
Description: Conan component: glib-gio-unix-2.0
Version: 2.73.3
Libs: -L"${libdir1}" 
Cflags: -I"${includedir1}" 
Requires: gobject-2.0 gio-2.0
```

So while the Cflags entry for the generated `pc` file is the same as the official one, the `includedir` variable is not.

Reply:
I see. Might be worth opening an issue in conan client repo to ask if a more canonical pkgconfig file could be generated by `PkgConfigDeps` generator.